### ISSUE TITLE
Source WooCommerce: Fetch all user roles (Customers Stream)

### ISSUE THREAD

**Issue by Dimi727** [airbytehq/airbyte#31201]
### What area the feature impact?

None

### Relevant Information

Currently, we only get WooCommerce users of the type `Customer` from the WooCommerce [API endpoint](https://woocommerce.github.io/woocommerce-rest-api-docs/#customers).

I did not find any official documentation how to get all types of users, but [here](https://stackoverflow.com/questions/55543951/rest-api-woocommerce-not-showing-all-customers-on-get-customers) someone mentioned exactly the same problem.

I think the stream should by default fetch all user types. In our case we have subscribers, which are also our customers but not getting fetched. Maybe its possible to add an additional parameter?
![image](https://github.com/airbytehq/airbyte/assets/23521125/e6292248-6cb8-498e-adfd-d950c20dcdd8)

**Comment by ombhardwajj** [comment]
Can I work on this until LinkedIn Ads Source is not ready in low code? @natikgadzhi

**Comment by natikgadzhi** [comment]
@girarda @strosek cool if @ombhardwajj works on this? ;)

**Comment by natikgadzhi** [comment]
@ombhardwajj good with me, but WooCommerce is low-code with a bunch of integration tests. A good solution here would be to run WooCommerce locally with abctl + Builder with custom components. I posted a video on how to do that here: https://www.youtube.com/watch?v=z02RKrrKT7M — please start with that and let me know when you get things running!

**Comment by natikgadzhi** [comment]
@ombhardwajj anything I can help you with here? Is this in-progress?

**Comment by ombhardwajj** [comment]
@natik I am currently engaged in #44510 and airbytehq/airbyte-dbt-models#18
Give me some time , I shall start work on this soon!

**Comment by natikgadzhi** [comment]
@ombhardwajj still interested, or should I unassign you?

**Comment by ombhardwajj** [comment]
Yeah @natikgadzhi  I am still interested, I will start work on this today!

**Comment by ombhardwajj** [comment]
@natikgadzhi I need a sandbox account in order to test the changes. Otherwise you can unassign me as I don't have access to an account to test it.

**Comment by octavia-squidington-iii** [comment]
At Airbyte, we seek to be clear about the project priorities and roadmap. This issue has not had any activity for 180 days, suggesting that it's not as critical as others. It's possible it has already been fixed. It is being marked as stale and will be closed in 20 days if there is no activity. To keep it open, please comment to let us know why it is important to you and if it is still reproducible on recent versions of Airbyte.
